### PR TITLE
Tests for Broker Card

### PR DESCRIPTION
**Pull request by satyasekharcvb**
### What does this PR do?
Adds Tests for Broker Card Component

[ X] Tests for the proposed changes have been added/updated.
[ X] Code linting and formatting was performed.

### Functionality Before

NA

### Functionality After

NA

### CODE REVIEW

File: force-app/test/jest-mocks/lightning/uiRecordApi.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (pozil) [inline force-app/test/jest-mocks/lightning/uiRecordApi.js:1]: Is there a particular reason why you're not using the official stub for this?

File: force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            // Return a promise to wait for any asynchronous DOM updates. Jest
+            // will automatically wait for the Promise chain to complete before
+            // ending the test and fail the test if the promise rejects.
+            return flushPromises().then(() => {
+                const propertyEl = element.shadowRoot.querySelector(
+                    'lightning-record-form'
+                );
+                expect(propertyEl.fields).toEqual(BROKER_FIELDS_INPUT);
+                expect(propertyEl.recordId).toBe(BROKER_ID);
+                expect(propertyEl.objectApiName).toBe(OBJECT_API_NAME_INPUT);
```
Comment (pozil) [inline force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js:93]: No need to check the `objectApiName` attribute, it's static in the template.

File: force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            document.body.appendChild(element);
+
+            // Emit data from @wire
+            getRecordAdapter.emit(mockGetPropertyRecord);
+
+            return flushPromises().then(() => {
+                const propertyEl = element.shadowRoot.querySelector(
+                    'lightning-record-form'
+                );
+                expect(getFieldValue.mock.calls.length).toBe(1);
```
Comment (pozil) [inline force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js:69]: You can use `expect(getFieldValue).toHaveBeenCalled();`

File: force-app/test/jest-mocks/lightning/uiRecordApi.js (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/test/jest-mocks/lightning/uiRecordApi.js]: implemented using the official stub

File: force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js (hunk 2 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js]: made the changes as suggested

File: force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js (hunk 3 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js]: made the changes as suggested

**Comment by pozil** [comment]
All good. Thx.

**Pull request merged**